service/scheduler/quartz/src/main/java/org/eclipse/kapua/service/scheduler/trigger/quartz/TriggerServiceImpl.java

```diff
@@ -163,7 +167,6 @@ public Trigger create(TriggerCreator triggerCreator) throws KapuaException {
                 scheduler.scheduleJob(quarztTrigger);
             } catch (SchedulerException se) {
                 se.printStackTrace();
-                throw new RuntimeException(se);
```

Comment:
This swallows exception, why was this throw removed?

Reply:
This part is reverted in last commit.